src/CosmosDB/CosmosDB/Helpers/Constants.cs

```diff
@@ -56,6 +56,9 @@ internal static class Constants
         public const string NetworkAclBypassHelpMessage = "Whether or not Network Acl Bypass is enabled for this account for Synapse Link. Possible values include: 'None', 'AzureServices'.";
         public const string NetworkAclBypassResourceIdHelpMessage = "List of Resource Ids to allow Network Acl Bypass for Synapse Link.";
 
+        //Backup specific help messages
+        public const string BackupIntervalInMinHelpMessage = "The interval(in minutes) with which backup are taken (only for accounts with periodic mode backups)";
```

Comment:
Can you check what are the guidelines whether to have complete word: Minutes or is this ok ?